runnotebooks.sh

```diff
@@ -1,53 +1,56 @@
 #!/usr/bin/env bash
 
-echo "# Running Notebooks Tests" >> $GITHUB_STEP_SUMMARY
+# https://betterdev.blog/minimal-safe-bash-script-template/
+set -Eeuo pipefail
```

Comment:
Uch, I messed this up, because I did not try to run the script without any parameters. If $1 is unset, the script breaks. See https://github.com/AstarVienna/ScopeSim_Data/actions/runs/7405748794